src/data/full-quiz.js

```diff
@@ -1,4 +1,47 @@
 const fullQuiz = [
+  
+    {
+    Question: "With SQL, how do you select all the records from a table named 'Persons' where the value of the column 'FirstName' starts with an 'a'?",
+    Answer: "SELECT * FROM Persons WHERE FirstName LIKE ‘a%’",

```

Comment:
For the answer and the three distractors, we should include semicolons at the end of those commands because it is important in SQL.